templates/components/password.build.hbs

```diff
@@ -1,19 +1,16 @@
 <input
   type="password"
   class="form-control"
-  :class="{ disabled: autogenerate }"
```

Comment:
If you're making this change, the following changes should probably be made too:

**Relevant to this PR**

https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/css/form.css#L276 Add `.form-control[readonly]` to selector

**Irrelevant to this PR**

https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/css/form.css#L498-L505 Add `.select-proxy-display>.hidden-select[readonly]~.icon` and `.select-proxy-display>.hidden-select[ready]~.select-value-proxy` to the selectors.